Description:
 Disable dead code in utility/Timer.cc, as it is causing build failures on some
 architectures.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: yes
Last-Update: 2010-03-27

--- a/utility/Timer.cc
+++ b/utility/Timer.cc
@@ -77,6 +77,8 @@
 
 // ---
 
+#if 0 /* dead code */
+
 Utility::TimebaseTimer::TimebaseTimer ()
 {
   start_tick = Value ();
@@ -185,3 +187,5 @@
   
   return per_second;
 }
+
+#endif
--- a/utility/Timer.hh
+++ b/utility/Timer.hh
@@ -97,7 +97,9 @@
     struct tms m_times;
   };
 #endif
-  
+
+#if 0 /* dead code */
+
   class TimebaseTimer
   {
   public:
@@ -138,6 +140,8 @@
     T m_timer;
     std::string m_text;
   };
+
+#endif
   
 } // end namespace utility
 
